beekeeper-package/pom.xml

```diff
@@ -17,28 +17,9 @@
     <module>beekeeper-assembly-path-scheduler-apiary</module>
   </modules>
 
-  <profiles>
-    <profile>
-      <id>snapshot-profile</id>
-      <activation>
-        <activeByDefault>true</activeByDefault>
-      </activation>
-      <properties>
-        <docker.repo>${artifactory.docker.snapshot.repository.id}.${artifactory.base}</docker.repo>
-      </properties>
-    </profile>
-    <profile>
-      <id>release-profile</id>
-      <activation>
-        <property>
-          <name>performRelease</name>
-          <value>true</value>
-        </property>
-      </activation>
-      <properties>
-        <docker.repo>${artifactory.docker.release.repository.id}.${artifactory.base}</docker.repo>
-      </properties>
-    </profile>
-  </profiles>
+  <properties>
+    <docker.org></docker.org>
```

Comment:
Should this be something other than null?

Reply:
Removed from here, this will be in eg-oss-parent

Reply:
Ohhh okay :)